### PR TITLE
Include Babylon Covenant Committee endpoints

### PR DESCRIPTION
**Pull request by filippos47**

### CODE REVIEW

File: bbn-test-4/covenant-committee/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Covenant Committee
+
+The Covenant Committee comprises entities operating
+[Covenant Signer](https://github.com/babylonchain/covenant-signer) daemons.
+
+For this testnet, the Committee Committee has 9 members, 3 of which are operated
+by the Babylon Foundation. The Babylon-operated endpoints are listed below:
```
Comment (vitsalis) [inline bbn-test-4/covenant-committee/README.md:7]: We should list all endpoints.

File: bbn-test-4/covenant-committee/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Covenant Committee
```
Comment (vitsalis) [inline bbn-test-4/covenant-committee/README.md:1]: Should we add a reference to this file on `bbn-test-4/README.md`?

Comment (filippos47) [reply bbn-test-4/covenant-committee/README.md:1]: Done!

File: bbn-test-4/covenant-committee/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+The Covenant Committee comprises entities operating
```
Comment (vitsalis) [inline bbn-test-4/covenant-committee/README.md:3]:
```suggestion
The Covenant Emulation Committee comprises entities operating
```

File: bbn-test-4/covenant-committee/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+[Covenant Signer](https://github.com/babylonchain/covenant-signer) daemons.
+
+For this testnet, the Committee Committee has 9 members, 3 of which are operated
```
Comment (vitsalis) [inline bbn-test-4/covenant-committee/README.md:6]:
```suggestion
For this testnet, the Committee Emulation Committee has 9 members, 3 of which are operated
```

File: bbn-test-4/covenant-committee/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+by the Babylon Foundation. A full list of the endpoints follows below, grouped
+by the operating entity.
```
Comment (vitsalis) [inline bbn-test-4/covenant-committee/README.md:8]: Let's state that the quorum and current members can be found in the parameters and link there?

**Review by vitsalis — APPROVED** [review]
minor

File: bbn-test-4/covenant-committee/README.md (hunk 5 above)
Comment (filippos47) [reply bbn-test-4/covenant-committee/README.md:8]: Added

**Pull request merged**